website/src/components/Changelog/GUI.tsx

```diff
@@ -14,11 +14,15 @@ export default function GUI({ title }: { title: string }) {
   return (
     <Entries href={href} arches={arches} title={title}>
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This must not be done when the issue's PR merges. */}
-      <Unreleased>
+      <Unreleased></Unreleased>
+      <Entry version="1.3.9" date={new Date("2024-10-09")}>
         <ChangeItem enable={title === "Linux GUI"} pull="6987">
           Fixes a crash on startup caused by incorrect permissions on the ID file.
         </ChangeItem>
-      </Unreleased>
+        <ChangeItem enable={title === "Windows"}>
+          This is a maintenance release with no user-facing changes.
+        </ChangeItem>
```

Comment:
We could probably make this default of `Entry` if we detect that there are no other children?

AFAIK, the `enable` thing isn't really idiomatic TSX. I would have written this as:

```tsx
{title == "Linux GUI" && <ChangeItem>...</ChangeItem>}
```

This works because `false`-ish values by default don't get rendered by React.

See https://react.dev/learn/conditional-rendering#logical-and-operator-.

If we do this, then it should be pretty simply to make a default child that says "maintenance release if there are no other children.